src/components/Windows/LoadingWindow/LoadingWindow.ts

```diff
@@ -15,11 +15,13 @@ export class LoadingWindow extends BaseWindow {
 	}
 
 	open(message?: string) {
+		//new Audio('/audio/click5.ogg').play()
```

Comment:
Can we remove the comments in this file and replace them with `// TODO: Add sounds`?